app/views/projects/_form_0.html.erb

```diff
@@ -219,11 +219,13 @@
                 # "(GPL-2.0+ WITH Bison-exception-2.2)".  See:
                 # https://spdx.org/spdx-specification-21-web-version
                 # Errors here do not cause *security* issues, so we're not
-                # too strict.
+                # too strict, but we want to check inputs so that our
+                # data is much more likely to be accurate.
                 f.text_field :license,
                              class: 'form-control', hide_label: true,
                              pattern: '(|[()A-Za-z0-9\-.+ ]*)',
```

Comment:
Is this pattern correct? What's that pipe doing there at the second character position? Perhaps this should be:

```
([()A-Za-z0-9\-.+ ]*)

Reply:
You're right.  I think I intended to end with "+" not "*", and the "|" allowed the empty response.  But since it's "*", may as well simplify things.

HOWEVER: This really needs to be a new issue.  You're commenting on an issue that was merged last year, so this discussion is invisible to most people.  Please continue the discussions in an open issue, such as: https://github.com/coreinfrastructure/best-practices-badge/issues/1226

Reply:
You're right.  I think I intended to end with "+" not "*", and the "|" allowed the empty response.  But since it's "*", may as well simplify things.

HOWEVER: This really needs to be a new issue.  You're commenting on an issue that was merged last year, so this discussion is invisible to most people.  Please continue the discussions in an open issue, such as: https://github.com/coreinfrastructure/best-practices-badge/issues/1226

See also:
https://github.com/coreinfrastructure/best-practices-badge/pull/1312